src/app-layout/__integ__/toolbar-tooltips/app-layout-toolbar-drawer-trigger-tooltip.test.ts

```diff
@@ -0,0 +1,165 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import createWrapper from '../../../../lib/components/test-utils/selectors/index';
+import { drawerIds as drawerIdObj } from '../../../../lib/dev-pages/pages/app-layout/utils/drawer-ids';
+import { VISIBLE_MOBILE_TOOLBAR_TRIGGERS_LIMIT } from '../constants';
+import { setupTest } from '../utils';
+
+const wrapper = createWrapper().findAppLayout();
+const drawerIds = Object.values(drawerIdObj);
+
+describe('refresh-toolbar', () => {
+  //using a theme variable below that will be set when iterating over ['refresh-toolbar', 'refresh']
+  const theme = 'refresh-toolbar';
+  const mobileDrawerTriggerIds = drawerIds.slice(
+    0,
+    VISIBLE_MOBILE_TOOLBAR_TRIGGERS_LIMIT + (theme === 'refresh-toolbar' ? 1 : 0)
```

Comment:
this condition will always be true, since the const above alway has the same value

Reply:
yes. but that will change shortly as refresh is also getting tooltips